### ISSUE TITLE
Add scrollbars to Option dialog

### ISSUE THREAD

**Issue by gitgordon** [notepad-plus-plus/notepad-plus-plus#825]
Hello,

as you can see in the screenshot, the listbox at the left side on the Option dialog has no scrollbars. So you can not read the complete option headline in some languages (here the german translation).

![2015-08-29_12h23_02](https://cloud.githubusercontent.com/assets/13393665/9561537/08965276-4e4a-11e5-9c3f-05e4177069b2.png)

Thank you
Gordon

**Comment by h-h-h-h** [comment]
:+1:
It would be a first solution. Of course, a listbox wide enough that in turn would let the scrollbars disappear would be the best.

**Comment by ewerybody** [comment]
On the other hand: Why does the german translation have this over specific wording? OK admitted: Its  hard to come up with shorter wording for "recent files history" :D
But "Delimiter" should be just "Trennzeichen" bam!

**Comment by xylographe** [comment]
`Datei/Vorherige Dateien`? (wie z.B. in PhotoFiltre)

**Comment by Ach3r0n** [comment]
I oppose against adding a scrollbar to the listbox, but I support making the listbox wider. Also the dutch translation for "recent files history" doesn't fit inside the current listbox.

P.S. I would also like some support for pull request #554 to fix some truncated text strings.

**Comment by h-h-h-h** [comment]
@Ach3r0n:
> I oppose against adding a scrollbar to the listbox, but I support making the listbox wider.

What if some string in some language will be too long again? A scrollbar appearing only then is appropriate. Someone can address the issue of a scrollbar appearing at a later time, but at least the full strings would be usable in the mean time.

**Comment by MetaChuh** [comment]
we are closing this issue due to its age and multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html).
if your issue persists, and it is still of relevance to you, please feel free to file a new issue.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)
